packages/rancher-monitoring/templates/crd-template/templates/jobs.yaml

```diff
@@ -20,28 +20,28 @@ spec:
         runAsNonRoot: true
         runAsUser: 1000
       initContainers:
-        - name: delete-crds
+        - name: set-preserve-unknown-fields-false
           image: {{ template "system_default_registry" . }}{{ .Values.image.repository }}:{{ .Values.image.tag }}
           imagePullPolicy: IfNotPresent
           command:
-          - /bin/kubectl
-          - delete
-          - --ignore-not-found=true
-          - -f
-          - /etc/config/crd-manifest.yaml
-          volumeMounts:
-            - name: crd-manifest
-              readOnly: true
-              mountPath: /etc/config
+          - /bin/sh
+          - -c
+          - >
+            {{- range $path, $_ :=  (.Files.Glob  "crd-manifest/**.yaml") }}
+            {{- $crd := get (get ($.Files.Get $path | fromYaml) "metadata") "name" }}
```

Comment:
Helm code that basically says "look at each file in crd-manifest that ends with .yaml and parse it as a YAML doc. Then pull out `metadata.name`'s value and put it in the variable $crd."

e.g. anywhere that says `{{ $crd }}` below would look like `prometheus.monitoring.coreos.com`.